src/output.cpp

```diff
@@ -341,7 +341,7 @@ void Output::ErrorStr(std::string const& err) {
 		std::cout << "EasyRPG Player will close now.";
 #if defined (GEKKO) || defined(__SWITCH__) || defined(_3DS)
 		// stdin is non-blocking
-		sleep(5);
+		Game_Clock::SleepFor(5s);
```

Comment:
I guess <unistd.h> might be removed for switch at least, wii likely.

Reply:
ok, also tidyed the file a bit